openprescribing/frontend/management/commands/measure_definitions/statinintensity.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "High-intensity statins",
+  "title": [
+    "Prescribing of high-intensity statins ",
+    "as a percentage of prescribing of all statins"
+  ],
+  "description": [
+    "Items of high-intensity statins ",
+    "as a percentage of items of all statins."
+  ],
+  "why_it_matters": [
+    "tba"
+  ],
+  "numerator_short": "High-intensity statin items",
+  "denominator_short": "All statins items",
+  "url": null,
+  "is_percentage": true,
+  "is_cost_based": false,
+  "low_is_good": false,
+  "numerator_columns": [
+    "SUM(items) AS numerator, "
+  ],
+  "numerator_from": "[ebmdatalab:hscic.normalised_prescribing_standard] ",
+  "numerator_where": [
+    "(bnf_code like '0212000Y0%AH' or bnf_code like '0212000B0%AB' or bnf_code like '0212000B0%AN' ",
+    "or bnf_code like '0212000B0%AC or bnf_code like '0212000B0%AP' or bnf_code like '0212000B0%AD' ",
+    "or bnf_code like '0212000AA%AA' or or bnf_code like '0212000AA%AB' or or bnf_code like '0212000AA%AC')"
```

Comment:
to be fair, i did it on a rickety train!  I need to change anyway as need to flip to low-intensity